lib/context_adapter_helper.js

```diff
@@ -562,10 +667,21 @@ function getExternalRequestId(response) {
  *  the obtained response
  */
 function sendThirdPartyRequest(updateContextRequest, callback) {
+  var payload = getThirdPartyPayload(updateContextRequest.contextAdapter.serviceDescriptor,
+    updateContextRequest.contextAdapter.operationDescriptor);
+
+  caLogger.debug(
+    updateContextRequest.contextAdapter.context,
+    'Sending ' + updateContextRequest.contextAdapter.serviceDescriptor.method + ' request to third party at: ' +
+      updateContextRequest.contextAdapter.serviceDescriptor.endpoint + ' with headers: ' +
+      'Fiware-Service: ' + updateContextRequest.headers['fiware-service'] + ' and ' +
+      'Fiware-ServicePath: ' + updateContextRequest.headers['fiware-servicepath'] +
+      ' and payload: ' + JSON.stringify(payload)
+  );
+
   request({
     method: updateContextRequest.contextAdapter.serviceDescriptor.method,
-    uri: updateContextRequest.contextAdapter.serviceDescriptor.endpoint +
-          (updateContextRequest.contextAdapter.serviceDescriptor.mapping.path || ''),
+    uri: updateContextRequest.contextAdapter.serviceDescriptor.endpoint,
```

Comment:
This literal object you are passing to request may be more readable if you define it in a variable above, instead of inlining the code inside the function call.